changelog/db.changelog-1.1.xml

```diff
@@ -0,0 +1,131 @@
+    <createTable tableName="paymentRequests">
+      <column name="paymentRequestId" type="INTEGER" autoIncrement="true">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="paymentRequestsId_pkey" />
+      </column>
+      <column name="agreementNumber" type="VARCHAR(20)" />
+      <column name="contractNumber" type="VARCHAR(10)" />
+      <column name="correlationId" type="VARCHAR(36)" /> <!-- interesting 1.23 rather tnan uuid -->
+      <column name="currency" type="VARCHAR(3)" />
+      <column name="deliveryBody" type="VARCHAR(4)" />
+      <column name="dueDate" type="VARCHAR(10)" />
+      <column name="invoiceNumber" type="VARCHAR(30)" />
+      <column name="marketingYear" type="INTEGER" />
+      <column name="referenceId" type="UUID" /> <!-- interesting 1.23 -->
+      <column name="schedule" type="VARCHAR(3)" />
+      <column name="sitiAgriInvoiceNumber" type="VARCHAR(20)" />
+      <column name="submitted" type="TIMESTAMP" />
+      <column name="value" type="INTEGER" />
+    </createTable>
+  </changeSet>
+  <changeSet author="Marc Templeton" id="4">
+    <createTable tableName="settlements">
+      <column name="settlementId" type="INTEGER" autoIncrement="true">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="settlementsId_pkey" />
+      </column>
+      <column name="paymentRequestId" type="INTEGER" >
+        <constraints nullable="false" />
+      </column>
+      <column name="detail" type="VARCHAR(255)" />
+      <column name="invalid" type="BOOLEAN" defaultValue="false" />
+      <column name="generated" type="BOOLEAN" defaultValue="false" />
+      <column name="frn" type="BIGINT" />
+      <column name="invoiceNumber" type="VARCHAR(30)" />
+      <column name="ledger" type="VARCHAR(2)" />
+      <column name="reference" type="VARCHAR(9)" /> <!-- based on PY1234567 -->
+      <column name="settled" type="BOOLEAN" defaultValue="false" />
+      <column name="settlementDate" type="TIMESTAMP" /> <!-- interesting in pay-pr this (named settled) is used for above bool -->
+      <column name="sourceSystem" type="VARCHAR(50)" />
+    </createTable>
+  </changeSet>
+  <changeSet author="Marc Templeton" id="5">
+    <addForeignKeyConstraint baseColumnNames="paymentRequestId" baseTableName="settlements" constraintName="fk_paymentRequests_paymentRequestId" referencedColumnNames="paymentRequestId" referencedTableName="paymentRequests" />
+  </changeSet>
+  <changeSet author="Marc Templeton" id="6">
+    <createTable tableName="schedules">
+      <column name="scheduleId" type="INTEGER" autoIncrement="true">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="schedulesId_pkey" />
+      </column>
+      <column name="settlementId" type="INTEGER" >
+        <constraints nullable="false" />
+      </column>
+      <column name="completed" type="TIMESTAMP" />
+      <column name="planned" type="TIMESTAMP" />
+      <column name="started" type="TIMESTAMP" />
```

Comment:
All `timestamps` do not include timezone, is this a pattern or when would we use with timezone?

Reply:
I can't think of a scenario where we'd specifically need the time zone.